### PR TITLE
Add guide to instruct the update of document and update sh file

### PR DESCRIPTION
**Pull request by Mingbo-Lee**
I found if run the `update_po.sh`, it got error;
```
(sf) limingbo@DESKTOP-mingbo:/mnt/d/WorkShop/secretflow/whitelist/secretflow/docs$ sh update_po.sh
: not foundh: 2:
Running Sphinx v5.3.0

Sphinx error:
not registered or available through entry point
] Error 2 [Makefile:20: gettext
: not foundh: 5:
po files has been updated. Please update po files in locales folder.
```
then, I think the sh file need updating to run well.

What's more, Inspired by https://github.com/secretflow/secretflow/pull/305, https://github.com/secretflow/secretflow/pull/556, https://github.com/secretflow/secretflow/pull/550, https://github.com/secretflow/secretflow/pull/551, I think there is a need to add guide document in CONTRIBUTING.md.
please TBR.

### CODE REVIEW

**Comment by zhouaihui** [comment]
Nice! LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,3 +46,21 @@ Pass `-S, --skip-string-normalization` to [black](https://github.com/psf/black)
 
 ## Git commit message style
 We stick to [Angular Style](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).
+
+
+## Documentation update and its multilingual version
+if you upudate the documentation files in `docs/`, you are supposed to update its multilingual version in `docs/locales/zh_CN/LC_MESSAGES`.
```
Comment (zhouaihui) [inline CONTRIBUTING.md:52]: `upudate` -> `update`

**Comment by Mingbo-Lee** [comment]
> Can you please add "fuzzy should be removed" to the markdown?

Ok, thank you for your suggestion.

**Review by zhouaihui — APPROVED** [review]
LGTM

**Pull request merged**